includes/class-wc-gateway-stripe.php

```diff
@@ -1288,7 +1288,13 @@ public function validate_account_statement_descriptor_field( $value, $max_length
 			! preg_match( $has_one_letter, $value ) ||
 			! preg_match( $no_specials, $value )
 		) {
-			throw new InvalidArgumentException( __( 'Customer bank statement is invalid. Statement should be between 5 and 22 characters long, contain at least single Latin character and does not contain special characters: \' " * &lt; &gt;', 'woocommerce-gateway-stripe' ) );
+			throw new InvalidArgumentException(
+				sprintf(
+					/* translators: %u Number of the maximum characters allowed */
+					__( 'Customer bank statement is invalid. Statement should be between 5 and %u characters long, contain at least single Latin character and does not contain special characters: \' " * &lt; &gt;', 'woocommerce-gateway-stripe' ),
```

Comment:
We are using the same validation failed message for both `Full bank statement` and `Shortened customer bank statement` fields, with the following message prefix: `Customer bank statement is invalid. Statement ...`, maybe we should have 2 different messages 🤷🏼?

On the other hand, the message is displayed above each of the fields so is clear which field has the error... feel free to dismiss this if you already evaluated the different messages.

Anyway, I just added an `a` to the message in the code suggestion. 

```suggestion
					__( 'Customer bank statement is invalid. Statement should be between 5 and %u characters long, contain at least a single Latin character and not contain special characters: \' " * &lt; &gt;', 'woocommerce-gateway-stripe' ),
```

Reply:
Good catch, I think it really makes sense to properly name each field so I addressed it in 41deb3b.